src/main/java/io/jenkins/plugins/analysis/core/model/ReportScanningTool.java

```diff
@@ -64,6 +65,21 @@ public String getPattern() {
         return pattern;
     }
 
+    /**
+     * Specify if file scanning should follow symbolic links.
+     *
+     * @param followSymlinks
+     *         the pattern to use
+     */
+    @DataBoundSetter
+    public void setFollowSymlinks(final boolean followSymlinks) {
+        this.followSymlinks = followSymlinks;
+    }
+
+    public boolean isFollowSymlinks() {
```

Comment:
I think Stapler requires the name to be `getFollowSymLinks` (or new `getNoFollowSymbolicLinks`)? Or does it work with `is` now? Maybe this has been fixed some time ago.